src/vs/editor/common/config/commonEditorConfig.ts

```diff
@@ -684,6 +684,11 @@ const editorConfiguration: IConfigurationNode = {
 			'default': EDITOR_DEFAULTS.contribInfo.codeLens,
 			'description': nls.localize('codeLens', "Controls whether the editor shows CodeLens")
 		},
+		'editor.cycleParameterHints': {
```

Comment:
Hm, maybe a better name which puts parameter hints first. My preference would be `editor.parameterHints.cycle` but that would require a refactoring of the existing `editor.parameterHints` setting, e.g. into `editor.parameterHints.enabled`.

Reply:
Ok, I've refactored the settings as such in my latest commit.